src/constants.js

```diff
@@ -6,4 +6,8 @@ export default {
     SHARED_WITH_ME: "Shared With Me",
     COMMUNITY_DATA: "Community Data",
     PATH_SEPARATOR: "/",
+    APPS_UNDER_DEV: "Apps under development",
+    FAV_APPS: "Favorite Apps",
+    MY_PUBLIC_APPS: "My public apps",
+    HPC: "High-Performance Computing",
```

Comment:
Not necessary for this PR, but I wonder if we should standardize on calling stuff HPC vs Agave?

Reply:
Correct. We also need to decide between `interactive` vs `VICE`